app/src/routes/event/[id]/+page.svelte

```diff
@@ -4,11 +4,17 @@
   import { formatDate, formatTime } from "$lib/utils";
   import { urlFor } from "$lib/sanity/image";
   import { signIn, signOut } from "@auth/sveltekit/client";
+  import type { RegistrationData } from '../../../models/registration-data.model';
```

Comment:
Hva med en @ alias på src her? Slipper lange ../../

Reply:
La til aliaser for models og components nå 😄

Reply:
Digg 👍